TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -96,7 +102,10 @@ class AllStopsTableViewController: UITableViewController {
             
         }
         
-        sectionIndexDictionary["#"] = numberBusStops
+        if (!allStops.isEmpty) {
+            sectionIndexDictionary["#"] = numberBusStops
```

Comment:
write clarifying comment on why set "#" to numberBusStops

Reply:
you should also remove `()` around the parans in the if statement. just `if !allStops.isEmpty`. This makes it cleaner :)